content/en/synthetics/private_locations/configuration.md

```diff
@@ -26,6 +26,9 @@ These configuration options for private locations can be passed as **parameters
 ```shell
 docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest --logFormat=json
 ```
+```shell
+docker run -d --restart always -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest --logFormat=json
+```
```

Comment:
Should this replace the old command? Having both might be a little confusing for users